firmware/thruna/BMS/src/app/app_imd.c

```diff
@@ -148,6 +160,11 @@ ImdCondition app_imd_getCondition()
     return condition;
 }
 
+uint8_t app_imd_pwmCounterTick()
```

Comment:
There is no need for this function anymore, you can replace all isntances of app_imd_pwmCounterTick() with io_imd_pwmCounterTick()